guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>The trained model <s>could</s> <u>can </u>now be saved using the <code>model.save(name_or_path)</code> command, which will save the model weights <u>and</u> create<s>s</s> an Esri Model Definition and Deep Learning Package zip. For example:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>